### PR TITLE
Added typings for jquery-treegrid

### PR DESCRIPTION
**Pull request by pmorelli92**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.

### CODE REVIEW

File: types/jquery-treegrid/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.5
+
+/// <reference types="jquery" />
+
+interface JQuery {
+    treegrid(options?: any): JQuery;    
+}
+
+declare var treegrid: JQueryStatic;
```
Comment (DanielRosenwasser) [inline types/jquery-treegrid/index.d.ts]: I'm not seeing this global variable when I try using the library.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
Thanks! Please remove the global `treegrid` unless I've missed something.

**Comment by pmorelli92** [comment]
@DanielRosenwasser done!

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Just as a heads up, your commits don't seem to be associated with your GitHub account. While this isn't technically a problem, you might care if you want more appropriate attribution. You can either make sure your GitHub account is associated with the email address you're using for your commits, or rebase and amend your commits to fix the author name and email.

If you don't care about any of this stuff, I can always just pull in the changes.

**Comment by DanielRosenwasser** [comment]
Uh oh - your rebase brought a bunch of merge commits. Try creating a new branch based off of DefinitelyTyped's `master`:

```
git branch add upstream https://github.com/DefinitelyTyped/DefinitelyTyped
git checkout upstream/master
git checkout -b jqueryTreegrid
```

and cherry-pick the following:

1. 500c02f
2. 6245eac

push the branch to your remote

```sh
git push origin jqueryTreegrid
```

then switch this PR's active branch to your `jqueryTreegrid` branch or open up a new PR.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
Problems with merge commit history.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has failing CI tests and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**